tests/test_carbon_handler.cairo

```diff
@@ -48,6 +48,7 @@ use carbon_v3::contracts::project::{
 // Constants
 
 const PROJECT_CARBON: u256 = 42;
+const MULT_ACCURATE_SHARE: u256 = 1_000_000;
```

Comment:
I guess this should be bigger to avoid too much rounding error.

Reply:
Will be done in next PR